project_euler/problem_14/sol2.py

```diff
@@ -26,7 +26,9 @@
 
 
 def collatz_sequence(n):
-    """Returns the Collatz sequence for n."""
+    """
+    Returns the Collatz sequence for n.
+    """
```

Comment:
```suggestion
    """Returns the Collatz sequence for n."""
```
One line docstring should as it was in the original solution.